roles/manage_touchstone_tools/tasks/start_db_stats.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Start ts pgsql-stat
+  ansible.builtin.shell: |
+    set -o pipefail
+    ts pgsql-stat \
```

Comment:
I feel that that the flags for database name, port and user should default to the system environment, and should only be explicitly set if the ansible variables are set.  That would match typical usage of apps that use libpq (e.g. psql).

I realize these roles are set to use root (`become: true`)... so maybe it is necessary to do this.  So then I'm wondering if they should match the naming schedule used in manage_dbserver. i.e. pg_database, pg_owner, etc.  I can't think of a reason why we'd want a different set of variable names.

Reply:
Take a look at how I set `start_db_stats` to see if this is sufficient for the first point. I changed the variable names to match the naming conventions used elsewhere

Reply:
:thinking: That's handling the two cases where all variables set, or no variables set right?  After some thought, I think I prefer how you were doing it originally.

If we are using `become: true` if the `pg_owner` is now set, for example, the it'll default to `root` (assuming nothing else in the environment overrides it), which isn't ideal.

We do want it to be run by root, because pidstat (or /proc) observes user permissions such that a non-root user can only gather it's own stats (or something like that.)